internal/database/conf_test.go

```diff
@@ -234,3 +234,159 @@ func TestSiteCreateIfUpToDate(t *testing.T) {
 		})
 	}
 }
+
+func seedDB(t *testing.T, ctx context.Context, s ConfStore) {
+	if _, err := s.SiteCreateIfUpToDate(ctx, nil, 0, `{"disableAutoGitUpdates": true, "auth.Providers": []}`, false); err != nil {
+		t.Fatal(err)
+	}
+
+	// The first call to SiteCreatedIfUpToDate will always create a default entry if their are no
+	// rows in the table yet and then eventually create another entry.
+	lastID := int32(2)
+
+	if _, err := s.SiteCreateIfUpToDate(ctx, &lastID, 1, `{"disableAutoGitUpdates": false, "auth.Providers": []}`, false); err != nil {
+		t.Fatal(err)
+	}
+
+	lastID = int32(3)
+
+	if _, err := s.SiteCreateIfUpToDate(ctx, &lastID, 1, `{"disableAutoGitUpdates": false, "auth.Providers": []}`, false); err != nil {
+		t.Fatal(err)
+	}
+
+	// By this point we have 4 entries instead of 3.
+}
```

Comment:
Why not use a loop? :)

```suggestion
func seedDB(t *testing.T, ctx context.Context, s ConfStore) {
	const dummyConfig = `{"disableAutoGitUpdates": true, "auth.Providers": []}`
	var lastID int32

	// The first call to SiteCreatedIfUpToDate will always create a default entry if their are no
	// rows in the table yet and then eventually create another entry.
	// After the loop we have 4 entries instead of 3.
	for i := 0; i < 3; i++ {
		siteConfig, err := s.SiteCreateIfUpToDate(ctx, nil, lastID, dummyConfig, false)
		if err != nil {
			t.Fatal(err)
		}
		lastID = siteConfig.ID
	}
}
```

Even if you don't use a loop, the `const dummyConfig` I would love to see, because otherwise it's noisy.

I'd also rename the function to `createDummySiteConfigs`. `seedDB` is awfully generic in a huge package like `internal/database`.